specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ManagedEnvironments.json

```diff
@@ -501,6 +717,99 @@
           "x-ms-secret": true
         }
       }
+    },
+    "Certificate": {
+      "description": "Certificate used for Custom Domain bindings of Container Apps in a Managed Environment",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Certificate resource specific properties",
+          "type": "object",
+          "properties": {
+            "password": {
+              "description": "Certificate password.",
```

Comment:
>password [](http://example.com/codeflow?start=42&length=8)

Do not return secret values in a GET operation. They should only be returned on POST : 

[https://armwiki.azurewebsites.net/rp_onboarding/process/property_design_best_practices.html#secrets](https://armwiki.azurewebsites.net/rp_onboarding/process/property_design_best_practices.html#secrets)

Reply:
@raosuhas - we do not return it in the GET operations. It is also marked as x-ms-secret. Do we need to do something else in swagger to indicate that it is  not returned on GET?

Reply:
Sry my bad .. I misread this. Please ignore my comment